website_legal_page/controllers/main.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import http
+from odoo.addons.website.controllers.main import Website
+
+
+class Legal(Website):
+
+    @http.route([
+        '/legal',
+        '/legal/<string:page>',
+    ],
+        auth='public',
+        type='http',
+        website=True,
+    )
+    def show(self, page='legal'):
```

Comment:
SEO-wise, if you have a `/legal` and a `/legal/legal` page that are the same, you get penalized.

I think you should better not add a default page and require it (404 error otherwise), or make it redirect to `/legal/legal` if `not page`.

Remember to add `**kwargs` so that we can add `?debug=assets` and so on.

Also, this controller's name should be `legal` instead of `show` IMHO.